include/open62541/server_pubsub.h

```diff
@@ -409,6 +411,9 @@ UA_Server_addDataSetField(UA_Server *server,
                           const UA_DataSetFieldConfig *fieldConfig,
                           UA_NodeId *fieldIdentifier);
 
+void UA_EXPORT
+UA_Server_updateDataSetField(UA_Server *server, const UA_NodeId dsf, UA_Boolean minor, UA_Boolean major);
```

Comment:
This function seems for me a little bit missunderstanding. Our API-Supports other "updateXX" functions, but these have a different purpose. Please think about a better name - maybe UA_Server_sendPubSubMetaData? In addition i can't get why you neeed the minor and major flags. The user should not decide if the version-number changes. It should be depended on PDS changes without user interaction.

Reply:
I renamed it to UA_Server_triggerDataSetFieldUpdate (https://github.com/wlkrm/open62541/blob/d5ce05ca1bda027fadb23789f277e7fa465900c1/include/open62541/server_pubsub.h#L418). The functions enables the user to inform the server that a DataSetField has changed. This allows to make multiple changes to the DataSetField, to avoid inconsistent states, e.g. if the user wants to make multiple changes "at once".  UA_Server_sendPubSubMetaData is not a suitable name imo, as the function does not really send MetaData.